README.md

```diff
@@ -11,31 +11,36 @@ production environment variables
 This gem helps pull down the environment variables and give you
 a little interface to view & compare them with (via a UI)
 
-No server; it's all done via a CLI. It's really just a wrapper around
-some commands around the [heroku cli](https://devcenter.heroku.com/articles/heroku-cli) that does some formatting and
-munging of the output so you don't have to copy it all to Excel
-
-This gem wraps the heroku CLI tool to generates a small temporary
-HTML file & opens it using [launchy](https://github.com/copiousfreetime/launchy)
+The heavy lifting is done via the [heroku platform-api gem](https://github.com/heroku/platform-api)
+to retrieve environment variables, format them in an HTML page, and
+generate a small HTML file and open it with [launchy](https://github.com/copiousfreetime/launchy)
 
 It automates a few things:
 - Hides environment variables that match across all environments
 - Puts them in alpha order
 
 It's primarily intended to compare pre-production ENV variables
+across similar/identical environments
 
 ## Installation
 
 ### Prerequisites
-- Install Heroku CLI
-  ```bash
-  brew tap heroku/brew && brew install heroku
-  ```
 
-- Login with Heroku CLI
-  ```bash
-  heroku login
-  ```
+```bash
+# Install Heroku CLI
+brew tap heroku/brew && brew install heroku
+# Login with Heroku CLI
+heroku login
+
+# Install the heroku oauth plugin to generate an OAuth token
+heroku plugins:install heroku-cli-oauth
+
+# Create a token to access your applications
+heroku authorizations:create -d "Platform API token for environment variables"
+
+# Set the Token from above to an environment variable
+export OAUTH_TOKEN=<token>
+```
```

Comment:
We should also add something like:
```
# Add permanently to `.bash_profile` (or whatever shell you use)
echo 'export OAUTH_TOKEN=<token>' >>~/.bash_profile
```

If not you get some gross errors once changing terminals. With that said, I think we should name the ENV VAR to something more explicit so it does not potentially conflict with other environment variables people might have set. `HEROKU_OAUTH_TOKEN_EC_GEM` 🤷‍♀️

Reply:
Let me add the echo/export as a note:
- If you have any experience with ENV vars, you already know this is how they work
- If you don't, doing things this way (temporarily, only this session) is least surprising behavior

I'm going to do some more research on the token name. The reason I chose OAUTH_TOKEN is because that's what the _actual_ gem uses. I think long term, it should pick up the platform-api token name, OR allow it to be passed in via an argument. Creating a 2nd ENV var that mirrors this (if they were using platform-api locally already) doesn't make much sense. But, it might make sense to just keep the token name "namespaced" to reduce conflicts. OAUTH_TOKEN is pretty generic

Reply:
The OAUTH_TOKEN is [read by the gem](https://github.com/heroku/platform-api/blob/cc3d0b911eed02d6144d26a6f79cd556092f1376/spec/support/helper_methods.rb#L22). I'm going to use that for compatibility rather than set a separate one

I'll add a comment to that text to add the echo to permanently add it